src/NuGet.Services.EndToEnd/NuGet.Services.EndToEnd.csproj

```diff
@@ -1,181 +1,183 @@
+    <Compile Include="Support\Clients\DotNetExeClient.cs" />
+    <Compile Include="Support\Clients\SearchServiceProperties.cs" />
+    <Compile Include="Support\Clients\V3IndexClient.cs" />
+    <Compile Include="Support\CommonCollection.cs" />
+    <Compile Include="Support\CommonFixture.cs" />
+    <Compile Include="Support\Configuration\ServiceDetails.cs" />
+    <Compile Include="ConnectivityTests.cs" />
+    <Compile Include="NuGetExeTests.cs" />
+    <Compile Include="RelistTests.cs" />
+    <Compile Include="SearchResultTests.cs" />
+    <Compile Include="Support\Clients\Clients.cs" />
+    <Compile Include="Support\Configuration\E2ESecretConfigurationReader.cs" />
+    <Compile Include="Support\Configuration\GalleryConfiguration.cs" />
+    <Compile Include="Support\Configuration\SearchServiceConfiguration.cs" />
+    <Compile Include="Support\ExceptionExtensions.cs" />
+    <Compile Include="Support\HttpRequestMessageException.cs" />
+    <Compile Include="Support\HttpResponseMessageExtensions.cs" />
+    <Compile Include="Support\PackageCreationContext.cs" />
+    <Compile Include="Support\RetryUtility.cs" />
+    <Compile Include="Support\SourceType.cs" />
+    <Compile Include="Support\CommandRunner.cs" />
+    <Compile Include="Support\CommandRunnerResult.cs" />
+    <Compile Include="Support\Clients\FlatContainerClient.cs" />
+    <Compile Include="Support\Clients\IGalleryClient.cs" />
+    <Compile Include="Support\Clients\NuGetExeClient.cs" />
+    <Compile Include="Support\Clients\RegistrationClient.cs" />
+    <Compile Include="Support\Clients\SimpleHttpClient.cs" />
+    <Compile Include="Support\PackageDirectory.cs" />
+    <Compile Include="Support\PackageType.cs" />
+    <Compile Include="Support\PushedPackagesCollection.cs" />
+    <Compile Include="Support\PushedPackagesFixture.cs" />
+    <Compile Include="Support\Clients\GalleryClient.cs" />
+    <Compile Include="Support\Package.cs" />
+    <Compile Include="Support\EnvironmentSettings.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.*.cs" />
+    <Compile Include="PushTests.cs" />
+    <Compile Include="Support\Clients\V2V3SearchClient.cs" />
+    <Compile Include="Support\TestSettings.cs" />
+    <Compile Include="Support\TestData.cs" />
+    <Compile Include="Support\TestDirectory.cs" />
+    <Compile Include="Support\XUnitExtensions\GalleryTestTheoryAttribute.cs" />
+    <Compile Include="Support\XUnitExtensions\SignedPackageTestFactAttribute.cs" />
+    <Compile Include="UnlistTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config">
+      <SubType>Designer</SubType>
+    </None>
+    <EmbeddedResource Include="Support\TestData\Icons\icon.png" />
+    <EmbeddedResource Include="Support\TestData\Icons\icon.jpg" />
+    <EmbeddedResource Include="Support\TestData\Readmes\readme.md" />

```

Comment:
What are the whitespace changes in this file? Should we undo them?

Reply:
Sure, it was defualt format in VS setting. I will change them back